server.js

```diff
@@ -1,53 +1,52 @@
-var debug = require('debug')('signalhubws')
+const uWS = require('uWebSockets.js')
+const debug = require('debug')('signalhubws')
 
-module.exports = function (WebSocketClass) {
-  var wss
-  var clients = []
-  var WebSocketServer = WebSocketClass || require('@clusterws/cws').WebSocketServer
+module.exports = function (ssl, ServerClass) {
```

Comment:
```suggestion
module.exports = function (WebSocketClass, ssl) {

ssl = ssl === undefined ? false : ssl
```

To avoid breaking the code that others may use.

Reply:
Sure yeah, I wanted to make it easier to use the default value so I made this argument second. But if this is being used as a library as opposed to an executable, then backward compatibility makes sense.